src/wazuh_db/wdb_parser.c

```diff
@@ -5914,9 +5915,13 @@ int wdb_parse_global_get_agents_by_connection_status(wdb_t* wdb, char* input, ch
     }
     connection_status = next;
 
+    /* Get node name */
+    next = strtok_r(NULL, delim, &savedptr);
```

Comment:
```suggestion
    node_name = strtok_r(NULL, delim, &savedptr);
```
You can simplify two lines into one.